kerl

```diff
@@ -1920,47 +2030,37 @@ list_add() {
     if [ -f "$KERL_BASE_DIR/otp_$1" ]; then
         while read -r l; do
             if [ "$l" = "$2" ]; then
-                return 1
+                return 0
             fi
         done <"$KERL_BASE_DIR/otp_$1"
-        echo "$2" >>"$KERL_BASE_DIR/otp_$1" || exit 1
+        echo "$2" >>"$KERL_BASE_DIR/otp_$1" || return 1
     else
-        echo "$2" >"$KERL_BASE_DIR/otp_$1" || exit 1
+        echo "$2" >"$KERL_BASE_DIR/otp_$1" || return 1
     fi
 }
 
 list_remove() {
     if [ -f "$KERL_BASE_DIR/otp_$1" ]; then
-        sed "$SED_OPT" -i -e "/^.*$2$/d" "$KERL_BASE_DIR/otp_$1" || exit 1
-    fi
-}
-
-list_has() {
```

Comment:
This function was also removed as "dead code". Lemme know if I should restore it.

Reply:
let's kill it